packages/wrangler/src/dev.tsx

```diff
@@ -381,34 +416,47 @@ export async function startDev(args: ArgumentsCamelCase<DevArgs>) {
 		async function getBindings(
 			configParam: Config
 		): Promise<CfWorkerInit["bindings"]> {
+			configParam.kv_namespaces ??= [];
+			configParam.durable_objects ??= { bindings: [] };
```

Comment:
why this? we don't want to mutate the config objects I think.

Reply:
Yeah, bit of a shortcut. I can rewrite that to be cleaner.